### PR TITLE
Added Mobile Stepper component

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-4373 .

#### Changes proposed in this Pull Request:

- Updated the kitchen sink component with MobileStepper example.
- Please verify MobileStepper component MD3 updates in this PR [MobileStepper](https://github.com/etn-ccis/blui-react-native-component-library/pull/395)

#### Screenshots / Screen Recording (if applicable)

Screenshot 2023-10-10 at 1 12 51 PM

#### To Test:

- yarn start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Review by huayunh — COMMENTED** [review]
any chance you can resolve that error in the CI build? Changes look good otherwise

**Comment by jvgreiner** [comment]
comment for how these should be tested. I'm currently in component library on branch release/md3 and the showcase is on feature/4373-mobile-stepper-md3 branch. When I run iOS I receive this error. Am I missing something to spin this up?

ERROR  Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `MobileStepperExample`.
    in MobileStepperExample (created by KitchenSink)

image

**Comment by manojleaton** [comment]
> comment for how these should be tested. I'm currently in component library on branch release/md3 and the showcase is on feature/4373-mobile-stepper-md3 branch. When I run iOS I receive this error. Am I missing something to spin this up?
> 
> ERROR Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
> 
> Check the render method of `MobileStepperExample`. in MobileStepperExample (created by KitchenSink)
> 
> image
You need to checkout to the branch `feature/4373-mobileStepper-MD3` in component library.

**Comment by jvgreiner** [comment]
> > comment for how these should be tested. I'm currently in component library on branch release/md3 and the showcase is on feature/4373-mobile-stepper-md3 branch. When I run iOS I receive this error. Am I missing something to spin this up?
> > ERROR Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
> > Check the render method of `MobileStepperExample`. in MobileStepperExample (created by KitchenSink)
> > image
> 
> You need to checkout to the branch `feature/4373-mobileStepper-MD3` in component library.

it was something with cache on my end, running now. thanks

**Review by jvgreiner — COMMENTED** [review]
looks good just run lint/prettier

**Review by ektaghag-eaton — APPROVED** [review]

File: md3-themes/MD3BluiLightTheme.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const MD3BluiLightTheme = {
         surfaceDisabled: 'rgba(26, 27, 30, 0.12)',
         onSurfaceDisabled: 'rgba(26, 27, 30, 0.38)',
         backdrop: 'rgba(45, 48, 56, 0.4)',
+        disabled: '#192024',
```
Comment (ektaghag-eaton) [inline md3-themes/MD3BluiLightTheme.ts:50]: We need to update colour in the dark theme.

**Review by ektaghag-eaton — CHANGES_REQUESTED** [review]

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**